### PR TITLE
Fixed some minor errors in openapi

### PR DESCRIPTION
**Pull request by xichengliudui**
/kind cleanup
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/kubectl/apply/parse/openapi.go [hunk 1]
```diff
@@ -147,7 +147,7 @@ type mapSchemaVisitor struct {
 	Result *proto.Map
 }
 
-// MergeMap implements openapi
+// VisiMap implements openapi
```
Comment (prksu) [inline pkg/kubectl/apply/parse/openapi.go]: some typo

~~VisiMap~~ VisitMap

Comment (xichengliudui) [reply pkg/kubectl/apply/parse/openapi.go]: updated

**Pull request merged**